config/routes.rb

```diff
@@ -12,6 +12,8 @@
   get '/cart', to: 'cart_items#index'
   get '/orders', to: 'orders#index'
   resources :orders, only: [:create,:show]
+  post 'twilio/voice' => 'twilio#voice'
+  post 'notifications/notify' => 'notifications#notify'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.